pkgs/applications/networking/cluster/kubernetes/default.nix

```diff
@@ -66,8 +67,8 @@ stdenv.mkDerivation rec {
     install -D build/pause/linux/pause -t $pause/bin
     installManPage docs/man/man1/*.[1-9]
 
-    # Unfortunately, kube-addons-main.sh only looks for the lib file in either the current working dir
-    # or in /opt. We have to patch this for now.
+    # Unfortunately, kube-addons-main.sh only looks for the lib file in either the
+    # current working dir or in /opt. We have to patch this for now.
```

Comment:
We do not enforce outdated line lengths.

Reply:
Right.  I made that change just to cause the reviewers to be set on the issue and because I wasn't sure if the tests would be run without a change to the package.

Reply:
You can trigger the tests with @ofborg tests kubernetes

Reply:
Fair enough.  I'll do that next time.

That said, the point of this PR was to get the tests to run automatically on changes, and starting them through a comment wouldn't have shown that the PR worked.